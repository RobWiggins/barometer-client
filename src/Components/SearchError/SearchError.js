import React from 'react';

export default class SearchError extends React.Component {
  render() {
    return (
      <div id="searchError">
        <p>
          oops! Heroku database wants my money to display! I shall never fall. I relish in trench warfare.
        </p>
      </div>
    );
  }
}
