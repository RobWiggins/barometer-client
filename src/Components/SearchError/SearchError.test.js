import React from 'react';
import ReactDOM from 'react-dom';
import SearchError from './SearchError';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('testing SearchError Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<SearchError />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders a SearchError component snapshot', () => {
    const wrapper = shallow(<SearchError />).find('#searchError');
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});