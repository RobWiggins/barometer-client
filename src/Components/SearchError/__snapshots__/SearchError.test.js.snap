// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing SearchError Component renders a SearchError component snapshot 1`] = `
<div
  id="searchError"
>
  <p>
    oops! Heroku database wants my money to display! I shall never fall. I relish in trench warfare.
  </p>
</div>
`;
